apps/app/src/app/shared/layout/side-nav/side-nav-content.component.ts

```diff
@@ -111,13 +111,13 @@ export class SideNavContentComponent {
 				{ label: 'Sheet', url: '/sheet' },
 				{ label: 'Skeleton', url: '/skeleton' },
 				{ label: 'Slider', url: '/slider' },
-				{ label: 'Sonner', url: '/sonner' },
+				{ label: 'Sonner (Toast)', url: '/sonner' },
```

Comment:
Sonner with hint that its the toast component

![CleanShot 2025-05-20 at 10 43 25](https://github.com/user-attachments/assets/759fd3f6-ea8a-42e3-963e-d9d1354a2d5c)